app/services/settings/authentication/upsert.rb

```diff
@@ -11,8 +11,6 @@ def self.call(settings)
       end
 
       def self.update_enabled_providers(value)
-        return if value.blank?
-
         enabled_providers = value.split(",").filter_map do |entry|
```

Comment:
`spec/requests/admin/configs_spec.rb` is failing when `value` is passed in is `nil`. I believe `.to_s` would fix the issue (nil will become an empty string) and the specs would now pass (I gave it a quick try locally as well).

I'm not sure if others would feel like a default value for the param would be better in this case though. The `.to_s` solution seems okay for me though 🙂 

```suggestion
        enabled_providers = value.to_s.split(",").filter_map do |entry|
```

Reply:
I totally meant to add to_s after citizen428's suggestion 🤦🏼‍♂️